src/wallet.js

```diff
@@ -1972,6 +1972,26 @@ const wallet = {
     return false;
   },
 
+  /**
+   * Get index of address.
+   * If address does not belong to the wallet (or was not generated yet) we return null, otherwise we return the index.
+   *
+   * @param {string} address Address to check
+   *
+   * @return {Number | null}
+   * @memberof Wallet
+   * @inner
+   */
+  getAddressIndex(address) {
+    const data = this.getWalletData();
+
+    if (this.isAddressMine(address, data)) {
```

Comment:
Don't all addresses in `data.keys` belong to the wallet?

Reply:
Yes. I just verify this to safely access `data.keys[address]`. Because if `!this.isAddressIsMine`, I don't have the key `address` in `data.keys`

Reply:
Ok. I just checked the `isAddressMine()` code and it is very simple. Thanks!